src/config/appConfig.js

```diff
@@ -0,0 +1,23 @@
+const appConfig = {
+  footer: {
+    links: [
+      { id: 10, value: 'about us', href: 'https://www.google.com', isOrigin: true },
+      { id: 23, value: 'our products', href: '/products', isOrigin: false },
+      { id: 36, value: 'join us', href: '/', isOrigin: false },
+      { id: 11, value: 'contact us', href: '/', isOrigin: false },
+      { id: 24, value: 'terms and conditions', href: '/', isOrigin: false },
+      { id: 37, value: 'live support', href: '/', isOrigin: false },
+      { id: 12, value: 'support', href: '/', isOrigin: false },
+      { id: 25, value: 'our privacy', href: '/', isOrigin: false }
+    ],
+    payments: [
+      { id: 8, value: 'amex', href: 'https://www.americanexpress.com/', isOrigin: true },
```

Comment:
`isOrigin` means the same origin, but here is external link

Reply:
Go it